### DISCUSSION TITLE
MLOps using GitHub

### DISCUSSION THREAD

**Discussion by ShrutiC-git** [github/india#103] [Satellite India 2021 Q&A]
If the intersection of ML and Ops is something you are interested in, GitHub can be a great way to start. If you have any questions about GitHub Workflow/GitHub Actions/DVC and how you can build ML Pipelines, feel free to drop them here, and let's talk about them!

**Comment by TheSuranaverse** [comment]
@ShrutiC-git 
A group for all to get notified for any `Official FRT Announcement`, `Discuss & Clear any Doubts`, `Learn & Complete` together.
One-stop place for daily `Events/Workshops` + `Hackathons` + `Internships/Jobs` + `Free Scholarship, Courses, Swags & Goodies` + `24/7 Live Chat and Mentors to help you with your Doubts`  + `24/7 Live Music` & much more...

Currently, we are helping all the `Future Ready Talent` Interns clear their doubts & queries on a personal basis either on chat or voice, whichever they prefer. We will also be assisting them on Projects & help them complete this internship 🤗

Link to join: https://discord.gg/drQ7ySCdX2
GitHub Solution Forum: https://github.com/github/india/discussions/353

Thank you 🙏
